src/resources/filters/normalize/pandoc3.lua

```diff
@@ -20,25 +20,22 @@ function parse_pandoc3_figures()
       Note = plain_figure_treatment,
       Figure = function(fig)
         if (#fig.content == 1 and fig.content[1].t == "Plain") then
-          
-          if #fig.content[1].content == 1 and fig.content[1].content[1].t == "Image" then
-            -- "pandoc 2 normalization"
-            local image = fig.content[1].content[1]
-            image.classes:extend(fig.classes)
-            for k, v in pairs(fig.attributes) do
-              image.attributes[k] = v
+          local forwarded_id = false
```

Comment:
Is it possible, or desirable, for us to protect ourselves from future regressions with some sort of testing coverage here?

Reply:
That makes sense.  Thanks for thinking about this and sharing your thoughts.